bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -157,8 +157,13 @@ object BagsState {
 
     // Successful AGS lookup while running (and not edited).  Apply the update
     // and move to IdleState.
-    private[ags] override def succeed(results: Option[AgsStrategy.Selection]): StateTransition =
-      (IdleState(k, Some(hash)), BagsManager.applyAction(k, results))
+    private[ags] override def succeed(results: Option[AgsStrategy.Selection]): StateTransition = {
+      val h = results.fold(hash) { r =>
+        if (r.posAngle === ctx.getPositionAngle) hash
```

Comment:
Agreed it's very trivial. It's just annoying to have these `Angle` comparisons in two different places, i.e. `BagsManager` and `ParallacticAngleControls`. If for any reason, we want to adjust the precision, then we will need to change in multiple places.

As this requires more thought, for now, I suppose we can just take the easy way out of doing this one liner, which I'll implement and get out this morning, but if we can think of enough cases where an `AlmostEqual` operation would be useful, this would be a nice thing to have.